### ISSUE TITLE
Deployment not possible anymore

### ISSUE THREAD

**Issue by pathob** [aservo/confapi-commons#157]
Hi,

I've just merged a pull request and the deployment to the Maven repository failed. There are currently migrations happening on the side of Sonatype, have you already started all the necessary steps for migration?

**Comment by pathob** [comment]
I think this is the issue

![Screenshot_20240614_124105_Fastmail](https://github.com/aservo/confapi-commons/assets/4580157/228c51b4-852b-4876-b5ad-7fda02b92ba6)

**Comment by brettaufheber** [comment]
Yes indeed, changes are actually already taking place here. These have already been made this changes for the LDAP-Crowd Adapter: https://gitlab.com/aservo/open-pool/open-source/ldap-crowd-adapter

If we migrate the confapi projects to GitLab, we will also make the update to the new Maven-Central plugin.
